### ISSUE TITLE
How to skip maxPercentIncrease for certain PR

### ISSUE THREAD

**Issue by zerofirework** [LironEr/bundlemon#179]
**Is your feature request related to a problem? Please describe.**
Sometimes, binary size increases are expected. How do we bypass the maxPercentIncrease limit for the PR that introduces such change?

**Describe the solution you'd like**
Specify a keyword in the description or a label to skip the failing bundlemon test.

**Describe alternatives you've considered**
Use `maxSize` instead since we can update the bundlemon settings in the same PR. But `maxPercentIncrease` shouldn't change

**Additional context**

**Comment by LironEr** [comment]
See here: https://github.com/LironEr/bundlemon/issues/115#issuecomment-1273363609

Forgot to document this ability, but let me know if this solves the issue for you

**Comment by zerofirework** [comment]
Yes. It's amazing! 

Quick question about the behavior, if I set the limit by max increase percent and I approve the change that increases the binary size, will the next PR fail the bundlemon CI again since the baseline didn't change and the binary size will stay high?

**Comment by LironEr** [comment]
yes, each approval is for a specific commit, the baseline will always be the target branch, once you merge the baseline will change.

**Comment by zerofirework** [comment]
I found two issues when trying this feature:

1. After I approve the change, the bundlemon CI job still fails. I was expecting it to pass.
2. There is a race condition in the UI between when the review button checks auth state and the auth state changes. If auth state changes after the review button checks the auth status, it will show me log in is required even if I have logged in.

**Comment by LironEr** [comment]
The approval applies to a specific commit that goes to GitHub and changes the current commit status from failed to success, re-running or a new commit will cause it to fail again.

We can think of a solution that checks the PR labels and auto-approves new jobs.

For now, you can do that bundlemon wont fail the pipeline
```
- name: Run BundleMon
   uses: lironer/bundlemon-action@v1
   with:
     fail-on-error: false

```
Then your job will always be successful, and if you want to verify the bundlemon status have a "required" status of the bundlemon commit status

**Comment by zerofirework** [comment]
Yeah, having a way to auto approve the jobs with a PR labels can be a great idea.

Do you mind elaborating what you meant by "have a "required" status of the bundlemon commit status"

Thanks for the reply!

**Comment by LironEr** [comment]
1. Setup [bundlemon github-integration](https://github.com/LironEr/bundlemon#github-integration), then on every commit in a PR bundlemon will add a commit status

![image](https://github.com/LironEr/bundlemon/assets/6559921/2422f89a-672c-4f42-9b31-abeef70b94ed)


2. In branch protection rules, make `BundleMon` a required status check
![image](https://github.com/LironEr/bundlemon/assets/6559921/dc869f02-8ff8-4836-800a-602f2d2a446a)

When one of the defined limits exceeds BundleMon will create a failed commit status, You can decide if you want to approve it via the BundleMon website. If you approve it will change the commit status to `success`
